packages/frontend/src/components/Home/RecentlyUpdatedPanel/index.tsx

```diff
@@ -87,6 +79,7 @@ const RecentlyUpdatedPanel: FC<Props> = ({ projectUuid }) => {
                 {
                     id: 'most-popular',
                     name: 'Most popular',
+                    icon: <IconStar size={20} />,
```

Comment:
I don't know if the spec changed, but when I worked on these @PriPatel said to remove the icons from the tabs